sites/platform/src/create-apps/app-reference.md

```diff
@@ -193,50 +193,152 @@ For more information, see how to [manage resources](/manage-resources.md).
 
 ## Relationships
 
-To access another container within your project, you need to define a relationship to it.
+To allow containers in your project to communicate with each other,
+you need to define relationships between them.
+You can define a relationship between an app and a service, or [between two apps](/create-apps/multi-app/relationships.md).
 
-{{% version/only "1" %}}
-![Relationships Diagram](/images/management-console/relationships.png "0.5")
-{{% /version/only %}}
+The quickest way to define a relationship between your app and a service
+is to use the service container's default endpoint.</br>
```

Comment:
"service container" might be confusing, since it also refers to a component in software architecture ([example with Symfony](https://symfony.com/doc/current/service_container.html))
```suggestion
is to use the service's default endpoint.</br>
```